GainsTracker.ClientNative/wwwroot/index.html

```diff
@@ -5,23 +5,73 @@
     <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover"
           name="viewport"/>
 
-    <title>GainsTracker.ClientNative</title>
+    <title>GainsTracker</title>
 
     <base href="/"/>
+    <link rel="icon" type="image/png" href="favicon.ico" />
 
     <link href="_content/GainsTracker.UI/css/site.css" rel="stylesheet"/>
     <link class="theme" href="_content/GainsTracker.UI/css/lightTheme.css" rel="stylesheet"/>
-
     <link href="GainsTracker.ClientNative.styles.css" rel="stylesheet"/>
+    
     <link href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css" rel="stylesheet"/>
+    
     <script src="_content/GainsTracker.UI/script/ElementManipulator.js" type="text/javascript"></script>
+    
+    <style>
+        .loading-container {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
```

Comment:
```suggestion
            height: 100vh;
            gap: 0.5rem;
```
Work with just rem or px for consistency (and rem is more accessibility approved! (if i remember correctly???))